deploy/penny-discord-bot-stack.yml

```diff
@@ -511,8 +513,4 @@ Outputs:
     Value: !GetAtt addCoinFunction.Arn
 
   dynamoDBArn:
-    Value: !GetAtt dynamoUserTable.Arn
-
-  
-
-
+    Value: !GetAtt dynamoUserTable.Arn
```

Comment:
```suggestion
    Value: !GetAtt dynamoUserTable.Arn

```
(note in case GH doesn't make it obvious: get rid of the "missing newline at end of file" symbol)

Reply:
I'm still not sure what you exactly meant but i think you meant to remove the last extra line?! i did that.

Reply:
```swift
await withCheckedContinuation { continuation in
    self.stateLock.withLockVoid { // prevent concurrent mutation
        self.loadWaiters.append(continuation)
    }
}
```
specifically here, i don't think any "concurrent mutations" can happen. 
I know there is a `await` there which means there is a suspension point, but i think the suspension point is triggered after the whole closure is executed, and is only used to wait for resumption of the continuation. A month or two ago I looked into the implementation of it but not through the right place, so didn't come to a proper conclusion.

Reply:
What I meant was "put the one of the trailing newlines back" 😰 See screenshot:
<img width="425" alt="Screenshot 2023-07-09 at 23 54 10" src="https://github.com/vapor/penny-bot/assets/1130717/2fb16c5b-5723-489e-a3e0-22405fd0d5f0">

Getting rid of all the _extra_ trailing newlines, definitely a good thing, I heartily approve - but you left no newline after `Arn` at the end of the last line. Git considers that bad form (and I happen to agree).